OpenKh.Kh2/Ard/Event.cs

```diff
@@ -856,8 +856,8 @@ private static void WriteVoices(MappingWriteArgs args)
 
         private static object ReadSetCameraData(MappingReadArgs args)
         {
-            List<SetCameraData.CameraKeys> AssignValues(SetCameraDataHeader header, IList<SetCameraData.CameraKeys> values) =>
```

Comment:
Why did you change the variable names here?

Reply:
Build was failing on NET Standard because of conflicting variable names.
"This variable already exists on this scope"